crates/ruff_python_ast/src/helpers.rs

```diff
@@ -17,26 +17,23 @@ use smallvec::SmallVec;
 
 use crate::call_path::CallPath;
 use crate::newlines::UniversalNewlineIterator;
-use crate::source_code::{Generator, Indexer, Locator, Stylist};
+use crate::source_code::{Generator, Indexer, Locator};
 use crate::statement_visitor::{walk_body, walk_stmt, StatementVisitor};
 
 /// Generate source code from an [`Expr`].
-pub fn unparse_expr(expr: &Expr, stylist: &Stylist) -> String {
-    let mut generator: Generator = stylist.into();
+pub fn unparse_expr(expr: &Expr, mut generator: Generator) -> String {
```

Comment:
For another PR: Should we move these methods to `Generator` instead, now that they always require a generator.

```
generator.unparse_expr(expr) 
// or

generator.generate_expr(expr)
```